packages/styles/search-field.css

```
@@ -0,0 +1,125 @@
+:root {
+  --search-input-border-color: var(--field-border-color);
+  --search-input-background-color: var(--field-background-color);
+  --search-input-border-focus-color: var(--field-border-color-focus);
+  --search-input-border-hover-color: var(--field-border-color-focus-hover);
+  --search-input-border-focus-glow-color: var(--field-border-color-focus-glow);
+  --search-input-error-border-color: var(--field-border-color-error);
+  --search-input-error-focus-border-color: var(
+    --field-border-color-error-hover
+  );
+  --search-input-error-focus-border-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --search-input-font-size: var(--field-font-size);
+  --search-input-font-color: var(--field-content-color);
+  --search-icon-color: var(--gray-90);
+}
+
+.cauldron--theme-dark {
+  --search-input-border-color: var(--field-border-color);
+  --search-input-background-color: var(--field-background-color);
+  --search-input-border-focus-color: var(--field-border-color-focus);
+  --search-input-border-hover-color: var(--field-border-color-focus-hover);
+  --search-input-border-focus-glow-color: var(--field-border-color-focus-glow);
+  --search-input-error-border-color: var(--field-border-color-error);
+  --search-input-error-focus-border-color: var(
+    --field-border-color-error-hover
+  );
+  --search-input-error-focus-border-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --search-input-font-color: var(--field-content-color);
+  --search-icon-color: var(--accent-light);
+}
+
+.Field__wrapper .TextFieldWrapper {
```

This is a wee bit too generic, this should probably be `SearchField`:
```suggestion
.SearchField :where(.TextFieldWrapper) {
```